favorites-service/src/main/java/com/yandex/market/favoritesservice/controller/api/FavoriteSellerApi.java

```diff
@@ -0,0 +1,63 @@
+package com.yandex.market.favoritesservice.controller.api;
+
+import com.yandex.market.favoritesservice.dto.request.FavoriteSellerRequest;
+import com.yandex.market.favoritesservice.dto.response.FavoritePreview;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.security.access.prepost.PreAuthorize;
+
+import java.util.UUID;
+
+public interface FavoriteSellerApi extends FavoritesApi {
+
+    @Operation(operationId = "createFavoriteSeller", summary = "Добавление магазина во вкладку избранное клиента")
+    @ApiResponse(responseCode = "201", description = "CREATED",
```

Comment:
```suggestion
    @ApiResponse(responseCode = "201", description = "Список избранных успешно создан",
```
Это только пример. Есть в этом смысл или нет, решать тебе, но у нас именно так было описано в продукт сервисе.